mathml/elements/mtable.json

```diff
@@ -0,0 +1,1062 @@
+{
+  "mathml": {
+    "elements": {
+      "math": {
```

Comment:
math -> mtable

Reply:
I literally [just said that](#discussion_r188647208).